spec/board_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/board.rb'
+
+RSpec.describe Board do
+
+  let(:board){@board = Board.new}
+
+  describe '#display' do
+    output = 1..9
+    it 'displays the board ' do
+     expect(board.display).to eql(output)
+    end   
+  end 
+
+  describe '#position_taken?' do 
+    let(:board){@board = Board.new}
+    it 'checks if there is space on the board' do
+      expect(board.position_taken?(1)).to eql(false)

```

Comment:
Let's test for both negative and positive cases.

```suggestion
      expect(board.position_taken?(1)).to eql(false)
      board.replace_var(0,'x')
      expect(board.position_taken?(1)).to eql(true)
```


 > Use your common sense: test positive and negative scenarios.

Reply:
done